server/controllers/operations.coffee

```diff
@@ -42,50 +42,73 @@ module.exports.delete = (req, res) ->
 
 module.exports.query = (req, res) ->
 
-    paramAccounts = req.body.accounts or [-1]
-
-    BankOperation.allFromBankAccounts paramAccounts, (err, operations) ->
-        if err?
-            res.send 500, error: 'Server error occurred while retrieving data'
+        paramAccounts = req.body.accounts or [-1]
```

Comment:
I don't know what squashing is. I will try to do it but not sure about what you want.